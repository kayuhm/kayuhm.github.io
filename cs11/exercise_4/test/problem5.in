4
3
7
10
33
16
63
4
42
