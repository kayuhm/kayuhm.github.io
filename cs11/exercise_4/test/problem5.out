5
10
4
DOES NOT EXIST
